src/webots/nodes/utils/WbObjectDetection.cpp

```diff
@@ -368,19 +368,19 @@ WbAffinePlane *WbObjectDetection::computeFrustumPlanes(const WbVector3 &devicePo
                                                        const double verticalFieldOfView, const double horizontalFieldOfView,
                                                        const double maxRange) {
   // construct the 4 planes defining the sides of the frustrum
-  double x = maxRange * tan(horizontalFieldOfView / 2.0);
-  double z = -maxRange;
-  double y = maxRange * tan(verticalFieldOfView / 2.0);
-  const WbVector3 topRightCorner = devicePosition + deviceRotation * WbVector3(x, y, z);
-  const WbVector3 topLeftCorner = devicePosition + deviceRotation * WbVector3(-x, y, z);
-  const WbVector3 bottomRightCorner = devicePosition + deviceRotation * WbVector3(x, -y, z);
-  const WbVector3 bottomLeftCorner = devicePosition + deviceRotation * WbVector3(-x, -y, z);
+  double z = maxRange * tan(verticalFieldOfView / 2.0);
+  double y = maxRange * tan(horizontalFieldOfView / 2.0);
+  double x = maxRange;
```

Comment:
Can't we make these const?:
```suggestion
  const double z = maxRange * tan(verticalFieldOfView / 2.0);
  const double y = maxRange * tan(horizontalFieldOfView / 2.0);
  const double x = maxRange;
```